static/css/devhub/new-landing/base.less

```diff
@@ -29,11 +30,20 @@
 @color-scheme-dark-background: #1d3770;
 @color-scheme-dark-items: #40d38d;
 
+// Buttons
+@color-button-default: #0568ba;
+@color-button-primary: #29c978;
+
```

Comment:
I split out the buttons from the core "theme" idea so the theming stuff is still in play just I saw that buttons could be treated somewhat independently of that.